nixos/modules/services/misc/docker-registry.nix

```diff
@@ -106,6 +108,16 @@ in {
       type = types.attrs;
     };
 
+    configFile = lib.mkOption {
+      default = null;
```

Comment:
Move the writeText here to scrap the nullable type and branch?

Reply:
Apparently, we can't do that because we're not allowed to use `cfg` in an options default (see https://github.com/NixOS/nixpkgs/actions/runs/8569274637/job/23484810758?pr=301815)

Reply:
Which commit was that? I can't grok that error message being related -- looks more like it needs a `literalExpression`, `defaultText`, or something like that, in order to render the manual

Reply:
I force pushed over it but it was https://github.com/NixOS/nixpkgs/compare/c0219cf0a9edd766933c0de30d8aa44a52c1ce12..3742ff98b45fe38f7164358a0a5cdddf18954fc9